.github/workflows/sdk-update-test.yml

```
@@ -0,0 +1,148 @@
+          FAUCET_SEED: ${{ secrets.PEREGRINE_FAUCET_SEED }}
+        run: |
+          yarn install
+          yarn run ts-node test.ts
+
+  core_features:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v1
+      - name: Get current package version
+        working-directory: code_examples/core_features
+        id: kiltprotocol_sdk
+        run: echo "::set-output name=current_tag::$(grep kiltprotocol/sdk-js package.json | awk -F \" '{print $4}')"
+      - uses: jacobtomlinson/gha-find-replace@v2
+        with:
+          find: '@kiltprotocol/sdk-js": "${{ steps.kiltprotocol_sdk.outputs.current_tag }}'
+          replace: '@kiltprotocol/sdk-js": "${{ github.event.inputs.client_payload.latestTag }}'
+          include: "code_examples/**"
+          exclude: "**/*.lock"
+          regex: true
+      - name: Test style conventions
+        working-directory: code_examples/core_features
+        run: |
+          yarn install
+          yarn lint
+
+      - name: Run tests
+        timeout-minutes: 60
+        working-directory: code_examples/core_features
+        env:
+          NODE_OPTIONS: --unhandled-rejections=strict
+          FAUCET_SEED: ${{ secrets.PEREGRINE_FAUCET_SEED }}
+        run: |
+          yarn install
+          yarn run ts-node run_core_features.ts
+
+   create_pull_request:
+     runs-on: ubuntu-latest
+     needs: [workshop, core_features]
+     if: success()
+     steps:
+       - uses: actions/checkout@v1
+       - name: Get current package version
+         working-directory: code_examples/core_features
+         id: kiltprotocol_sdk
+         run: echo "::set-output name=current_tag::$(grep kiltprotocol/sdk-js package.json | awk -F \" '{print $4}')"
+       - uses: jacobtomlinson/gha-find-replace@v2
+         with:
+           find: '${{ steps.kiltprotocol_sdk.outputs.current_tag }}'
+           replace: '${{ github.event.client_payload.latestTag }}'
+           include: "docs/develop/03_workshop/**"
+           regex: true
+       - uses: jacobtomlinson/gha-find-replace@v2
+         with:
+           find: '@kiltprotocol/sdk-js": "${{ steps.kiltprotocol_sdk.outputs.current_tag }}'
+           replace: '@kiltprotocol/sdk-js": "${{ github.event.client_payload.latestTag }}'
+           include: "code_examples/**"
+           exclude: "**/*.lock"
+           regex: true
+       - name: Regenerate yarn lock
+         working-directory: code_examples/core_features
+         run: |
+           yarn install
+           yarn lint
+       - name: Regenerate yarn lock
+         working-directory: code_examples/workshop
+         run: |
+           yarn install
+           yarn lint
+       - name: Create Pull Request
+         uses: peter-evans/create-pull-request@v4
+         with:
+           token: ${{ secrets.REPO_ACCESS_TOKEN }}
+           base: develop
+           delete-branch: true
+           title: '[Chore] Latest sdk is compatible with docs'
```

Something like this:

```suggestion
           title: '[Chore] Update SDK version to latest (still compatible with the current docs)'
```